docs/content/filtering.md

```diff
@@ -96,191 +134,285 @@ With `--ignore-case`
+define an rclone pattern list.
+
+### `--exclude-from` - Read exclude patterns from file ###
+
+Excludes path/file names from an rclone command based on rules in a
+named file. The file contains a list of remarks and pattern rules. 
+
+For an example `exclude-file.txt`:
 
     # a sample exclude rule file
     *.bak
     file2.jpg
 
-Then use as `--exclude-from exclude-file.txt`.  This will sync all
-files except those ending in `bak` and `file2.jpg`.
+`rclone ls remote: --exclude-from exclude-file.txt` lists the files on
+`remote:` except those named `file2.jpg` or with a suffix `.bak`. That is
+equivalent to `rclone ls remote: --exclude 'file2.jpg' --exclude '*.bak'`.
+
+This flag can be repeated. See above for the order filter flags are
+processed in.
+
+The `--exclude-from` flag is useful where multiple exclude filter rules
+are applied to an rclone command.
+
+`--exclude-from` should not be used with `--include`, `--include-from`,
+`--filter` or `--filter-from` flags.
+
+`--exclude-from` has no effect when combined with `--files-from` or
+`--files-from-raw` flags.
 
-This is useful if you have a lot of rules.
+`--exclude-from` followed by `-` reads filter rules from standard input. 
 
 ### `--include` - Include files matching pattern ###
 
-Add a single include rule with `--include`.
+Adds a single include rule based on path/file names to an rclone
+command.
 
-This flag can be repeated.  See above for the order the flags are
+This flag can be repeated. See above for the order filter flags are
 processed in.
 
-Eg `--include *.{png,jpg}` to include all `png` and `jpg` files in the
-backup and no others.
+`--include` has no effect when combined with `--files-from` or
+`--files-from-raw` flags.
 
-This adds an implicit `--exclude *` at the very end of the filter
-list. This means you can mix `--include` and `--include-from` with the
-other filters (e.g. `--exclude`) but you must include all the files you
-want in the include statement.  If this doesn't provide enough
-flexibility then you must use `--filter-from`.
+`--include` implies `--exclude **` at the end of rclone's internal
+filter list. Therefore if you mix `--include` and `--include-from`
+flags with `--exclude`, `--exclude-from`, `--filter` or `--filter-from`,
+you must use include rules for all the files you want in the include
+statement. For more flexibility use the `--filter-from` flag.
 
-### `--include-from` - Read include patterns from file ###
+Eg `rclone ls remote: --include '*.{png,jpg}'` lists the files on
+`remote:` with suffix `.png` and `.jpg`. All other files are excluded.
 
-Add include rules from a file.
+Eg multiple rclone copy commands can be combined with `--include` and a
+pattern-list.
 
-This flag can be repeated.  See above for the order the flags are
-processed in.
+    rclone copy /vol1/A remote:A
+    rclone copy /vol1/B remote:B
 
-Prepare a file like this `include-file.txt`
+is equivalent to:
+
+    rclone copy /vol1 remote: --include={A,B}/**
```

Comment:
This should be anchored, so `--include "/{A,B}/**"`

Reply:
Done.